crates/top/rerun_c/src/rerun.h

```diff
@@ -212,6 +212,64 @@ typedef struct {
     rr_component_batch* component_batches;
 } rr_data_row;
 
+/// Arrow-encoded data of a component batch partitioned into several runs of components.
+///
+/// This is essentially an array of `rr_component_batch` with all batches
+/// continously in a single array.
+typedef struct rr_partitioned_component_batch {
+    /// The component type to use for this batch.
```

Comment:
Is this the arrow datatype of the component?

Reply:
yes. Will clarify. The this is the datatype that is wrapped with `List` arrow datatype